cypress/helpers/randomUser.js

```diff
@@ -1,5 +1,16 @@
 const date = require('../helpers/date')
 
+module.exports = function () {
```

Comment:
What worries me about this function is that the call to it will generate the exact same user name and email multiple times if called in the same minute. 
The previous function added some random characters to avoid this issue.

I suggest adding some randomness to it so is not an issue if this function is called multiple times during the same minute.

Reply:
Completed all changes required.